iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/tree/api/Node.java

```diff
@@ -36,4 +38,11 @@ public interface Node {
    * @return the location of the node in the file
    */
   Location location();
```

Comment:
I just thought that maybe we should've converted `Location`s to `TextRange`s first, and then operate only with them. But maybe it's something to consider when reworking the Go AST

Reply:
Yes, let's think about it later.